package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-TAT",
+  "name": "engagesphere-test-design-masterclass-tat",
```

Não há necessidade alguma de mudar o nome do projeto.

```suggestion
  "name": "EngageSphere-Test-Design-Masterclass-TAT",
```